Calibration/HcalAlCaRecoProducers/src/AlCaIsoTracksProducer.cc

```diff
@@ -163,7 +162,7 @@ AlCaIsoTracksProducer::AlCaIsoTracksProducer(const edm::ParameterSet& iConfig) :
   tok_trigRes_  = consumes<edm::TriggerResults>(labelTriggerResults_);
   tok_genTrack_ = consumes<reco::TrackCollection>(labelGenTrack_);
   tok_recVtx_   = consumes<reco::VertexCollection>(labelRecVtx_);
-  tok_bs_       = consumes<reco::BeamSpot>(labelBS_);
+  tok_bs_       = consumes<reco::BeamSpot>(edm::InputTag("offlineBeamSpot"));
```

Comment:
Indeed, here it is hard coded when getting the EDM token:
https://github.com/cms-sw/cmssw/pull/8943/files#diff-437898eff887c85e588dfc2133baff8dR7